app/org/sagebionetworks/bridge/dynamodb/DynamoStudy.java

```diff
@@ -187,6 +189,30 @@ public void setUserProfileAttributes(Set<String> profileAttributes) {
     }
 
     /** {@inheritDoc} */
+    @DynamoDBMarshalling(marshallerClass = StringSetMarshaller.class)
```

Comment:
In short, it's needed. Without it and no other modifications, you get an error: 
[error] Test org.sagebionetworks.bridge.dynamodb.DynamoStudyDaoTest.crudOneStudy failed: com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBMappingException: Expected SS in value {S: ["phone","can_be_recontacted"],} when invoking public void org.sagebionetworks.bridge.dynamodb.DynamoStudy.setUserProfileAttributes(java.util.Set), took 0.123 sec
[error]     at com.amazonaws.services.dynamodbv2.datamodeling.unmarshallers.SSUnmarshaller.typeCheck(SSUnmarshaller.java:28)

I'm not sure what it wants, it may simply need this to work with sets rather than lists... ?